pickled.py

```diff
@@ -0,0 +1,69 @@
+import requests
```

Comment:
Good use of separate data fetching script that allows you to run a data analysis script separately from your data collection script.